en/concept/core_components.md

```diff
@@ -0,0 +1,91 @@
+# PX4 Core Components
+
+These "core components" define the core functionality of PX4-Autopilot.
+
+They are tested in continuous integration tests and are regularly monitored and tested by their maintainers.
+Core components are the first priority in stable point releases.
+
+## Core component criteria
+
+Each core component must have:
+
+- Maintainer
+- Tester
+
+## PX4 Core Components List
+
+For many of the core components that has a corresponding label in Github, you can click on them to view the relevant Issue/PRs.
+
+| Core component (category)                                                          | Sub component                        | Maintainer                                       |
+| ---------------------------------------------------------------------------------- | ------------------------------------ | ------------------------------------------------ |
+| [uORB (Architecture)][uORB (Architecture)]                                         |                                      |                                                  |
+| [Parameter (Architecture)][Parameter (Architecture)]                               |                                      |                                                  |
+| Realtime Clock (Architecture)                                                      |                                      |                                                  |
+| Work queue (Architecture)                                                          |                                      |                                                  |
```

Comment:
IMO all of these are subcomponets of "architecture", owned by either/both Beat and Daniel.